code/lib/manager-api/src/modules/channel.ts

```diff
@@ -40,6 +47,9 @@ export const init: ModuleFn<SubAPI, SubState> = ({ provider }) => {
       }
       provider.channel.emit(type, data, ...args);
     },
+    emitServerAction: (type, data, ...args) => {
+      provider.serverChannel.emit(type, data, ...args);
+    },
```

Comment:
Is this the same as?

```suggestion
    emitServerAction: (type, ...args) => {
      provider.serverChannel.emit(type, ...args);
    },
```

Reply:
No, not exactly. I choose to not dive too deep and simply mimic the api of `emit`.